### PR TITLE
Fix: _jsonInterfaceMethodToString export in types

### PR DESCRIPTION
**Pull request by zlumer**
Found a TypeScript export error in web3.js v1.x

`web3.utils.jsonInterfaceMethodToString` is [exported](https://github.com/web3/web3.js/blob/a7b5dea0e98c0b7742c2477c511cb16acc1f27c2/packages/web3-utils/src/index.js#L381) with underscore in JS. In TS types however the underscore is not present.

Steps to reproduce:

```typescript
import Web3 from "web3"

let web3 = new Web3()

console.log(`web3.utils.jsonInterfaceMethodToString: `, web3.utils.jsonInterfaceMethodToString)
// outputs: "web3.utils.jsonInterfaceMethodToString:  undefined"

console.log(`web3.utils._jsonInterfaceMethodToString: `, (web3.utils as any)._jsonInterfaceMethodToString)
// outputs: "web3.utils._jsonInterfaceMethodToString:  [Function: _jsonInterfaceMethodToString]"
```

This PR fixes this error in web3.js 1.x

### CODE REVIEW

**Comment by zlumer** [comment]
Can we merge this or is anything blocking atm?

**Comment by Muhammad-Altabba** [comment]
Thanks @zlumer for your contribution.
To merge this you need to get all the tests to pass. So, could you please investigate why there is one failing?

**Comment by zlumer** [comment]
From what I can see in the logs, the problem seems to be with the infura connection.

Not sure if this has something to do with a simple one-character typings change.

Do any of the other PRs have the same problem?

```
>>>>>>
HTTP:MAINNET getBlock
>>>>>>
Error: Failed to connect to Infura over websockets after 10 tries
    at getBlockWithRetry (D:\a\web3.js\web3.js\windows_test\basic_usage.js:31:15)
Error: Process completed with exit code 1.
```

**Comment by zlumer** [comment]
I just checked the Dependabot PRs (e.g. #5551) and they have the same error in the logs.

I suspect it may be the effect of The Merge and the testnet deprecation by Infura:
https://blog.infura.io/post/deprecation-timeline-for-rinkeby-ropsten-and-kovan-testnets

**Review by Muhammad-Altabba — APPROVED** [review]
Thanks @zlumer,
Yes, it seems to be an irrelevant issue in the pipeline. And I created an issue for this based on your comment. 
And I think this MR is good to go once we have additional approval from the team.

**Comment by Muhammad-Altabba** [comment]
Could you please update the CHAINLOG.md file?

**Comment by zlumer** [comment]
> Could you please update the CHAINLOG.md file?

Done 🙂

File: CHANGELOG.md
Comment (jdevcs) [inline CHANGELOG.md]: Kindly rebase your branch and put this under unreleased `1.8.2` . Thanks

Comment (zlumer) [reply CHANGELOG.md]: Done.

Should I squash commits now or during the PR merge?

Comment (luu-alex) [reply CHANGELOG.md]: squash commits will happen during pr merge

**Review by luu-alex — APPROVED** [review]

**Review by avkos — APPROVED** [review]

**Pull request merged**
